SEO/README.md

```diff
@@ -44,6 +44,27 @@ The high-priority-queue responds to changes in ACE Knowledge Core, eg. in guides
 
 One example of a change running in the low-priority-queue is statistics that update the  <wbr>ACE Knowledge's top lists. Changes detected in the low-priority-queue are indexed once a day, outside of office hours, to reduce the load on ACE knowledge's servers. The low-priority-queue is run only if the high-priority queue is empty.
 
+**Overview ACE Knowledge events**  
+How quickly are modifications made in ACE Knowledge Admin reflected in the SEO Service?
+
+* Widgets
+  * Deleted: `IMMEDIATE`
+  * Changed name: previous urls removal `IMMEDIATE`, new urls `HIGH`
+  * Changed SEO status to Off: `IMMEDIATE`
+  * Changed SEO status to not Off: `HIGH`
+  * Other changes: `HIGH`
+* Implementations
+  * Any change: `HIGH` - will be sent for any widget with SEO enabled using the implementation
```

Comment:
Jag kände inte till det att även dessa skickades. Men du har skrivit koden så jag litar på dig ang denna.

Hur funkar `DependencyEntityIds` för Implementation`? Widgetens ÈntityId skickas med vid registreringen men kommer även Implementation Id där? Annars får inte någon träff på `SeoPage` som borde uppdateras.

Reply:
https://github.com/Humany/humany-core/blob/master/src/Infrastructure/KnowledgeBase/Handlers/SeoNotifyGuideHandler.cs#L102
Det som skickas är alla seo-enablade widgets som använder Site/Implementation i fråga
`ChangeNotification { Tenant = application.Subdomain, ChangedEntityIds = seoWidgets.Select(o => o.Id).ToList() }`

Reply:
Eftersom `widgetId` skickas till SEO-service så kommer det att fungera fint.

Reply:
Skulle det vara tydligare om man skrev? "will be sent for all widgets that are using the implementation and have SEO enabled"
Eller "within" istället för "that are using"?

Reply:
Ja, det första låter bra! (Jag har samma tendens att korta ner onödigt mycket i både naturligt språk som i kod :) )